docs/reference/cache.rst

```diff
@@ -66,6 +66,31 @@ or you can use the `Response` object:
         return Response::create(sprintf("your name is %s", $user->getUsername()))->setTtl(0)->setPrivate();
     }
 
+Cache Keys
+::::::::::
+
+The ``SonataCacheBundle`` needs cache keys in order to find cached
+blocks. By default cache keys consist of the ``block_id`` and the ``updated_at``
+timestamp. Because these values change on every call from the Twig Helper,
+it is mandatory to overwrite the ``getCacheKeys`` function in your custom block class:
+
+.. code-block:: php
+
+    <?php
+
+    class CachedBlock extends AbstractBlockService implements BlockServiceInterface
```

Comment:
```suggestion
    final class CachedBlock extends AbstractBlockService
```

please `final` and `AbstractBlockService` already implements `BlockServiceInterface`